src/test/atcoder_test.py

```diff
@@ -0,0 +1,205 @@
+import unittest
+from unittest.mock import patch, MagicMock
+import json
+from src.scrape_up.atcoder import Atcoder
+
+
+class TestAtcoder(unittest.TestCase):
+    """
+    | Methods           | Details                                                                            |
+    | ----------------- | ---------------------------------------------------------------------------------- |
+    | `.get_profile()`  | Returns the user data in json format.                                              |
+    | `get_contests()`  | Returns future_contests , past_contests , skill_tests etc in json format.          |
+
+    """
+    def setUp(self):
+        # Initialize an Atcoder instance before each test method
+        self.user = "testuser"
+        self.atcoder = Atcoder(user=self.user)
+        
+    @patch('requests.get')
+    def test_get_profile(self, mock_get):
+        try:
+            # Mock response for get_profile method
+            mock_response = MagicMock()
+            mock_response.text = '''
+                <html>
+                    <table class="dl-table">

```

Comment:
These are some unnecessary changes, you just have to instantiate the class and call the methods. Then validate if it is returning the desired values or not. 

You can check other file to see how the test files are formatted.

Reply:
Sure, I will